system-int-test/test/cmr/system_int_test/ingest/collection_ingest_test.clj

```diff
@@ -604,15 +606,29 @@
         response (ingest/ingest-concept coll-map {:accept-format :json})]
     (is (= 201 (:status response)))))
 
+(deftest ingest-higher-than-accepted-umm-version
+  (let [accepted-version (config/ingest-accept-umm-version)
+        _ (side/eval-form `(config/set-ingest-accept-umm-version! "1.8"))
+        coll-map {:provider-id  "PROV1"
+                  :native-id    "umm_json_coll_V1"
+                  :concept-type :collection
+                  :format       "application/vnd.nasa.cmr.umm+json;version=1.9"
+                  :metadata     "{\"foo\":\"bar\"}"}
+        response (ingest/ingest-concept coll-map {:accept-format :json})
+        _ (side/eval-form `(config/set-ingest-accept-umm-version! ~accepted-version))]
+    (is (= 400 (:status response)))
+    (is (= [(str "umm version is higher than ingest-accept-umm-version\"" "1.8" "\"")]
+           (:errors response)))))
+
 (deftest ingest-invalid-umm-version
   (let [coll-map {:provider-id  "PROV1"
                   :native-id    "umm_json_coll_V1"
                   :concept-type :collection
-                  :format       "application/vnd.nasa.cmr.umm+json;version=9000.1"
```

Comment:
The part that validates that the version is actually a valid UMM version is later in the process. it actually goes into the the file system to see if that schema file exists.  My thought is:
If the version is higher than the configured accepted version, it should be rejected right away without wasting the work to go in to find out that version doesn't exist.